sdk/samples/iot/docs/how_to_iot_hub_realtek_amebaD.md

```diff
@@ -0,0 +1,264 @@
+# How to Setup and Run Azure SDK for Embedded C IoT Hub Client on Realtek AmebaD
+
+- [How to Setup and Run Azure SDK for Embedded C IoT Hub Client on Realtek AmebaD](#how-to-setup-and-run-azure-sdk-for-embedded-c-iot-hub-client-on-realtek-amebad)
+  - [Introduction](#introduction)
+    - [What is Covered](#what-is-covered)
+  - [Prerequisites](#prerequisites)
+  - [Setup and Run Instructions](#setup-and-run-instructions)
+  - [Troubleshooting](#troubleshooting)
+  - [Contributing](#contributing)
+    - [License](#license)
+
+## Introduction
+
+This is a "to-the-point" guide outlining how to run an Azure SDK for Embedded C IoT Hub telemetry sample on an Realtek AmebaD development board. The command line examples are tailored to Windows and Linux environments.
```

Comment:
`The command line examples are tailored to Windows and Linux environments.`
Is this true? This looks like an Arduino project